setup.py

```diff
@@ -12,74 +12,46 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import absolute_import
-from __future__ import print_function
-
-import sys
 from glob import glob
-from os.path import basename
-from os.path import splitext
-
-from setuptools import find_packages
-from setuptools import setup
-from setuptools.command.test import test as TestCommand
-
-
-# Allows one to simply > python3 setup.py test
-class PyTest(TestCommand):
-    user_options = [('pytest-args=', 'a', "Arguments to pass to py.test")]
+from os.path import basename, splitext
 
-    def initialize_options(self):
-        TestCommand.initialize_options(self)
-        self.pytest_args = []
+from setuptools import find_packages, setup
 
-    def finalize_options(self):
-        TestCommand.finalize_options(self)
-        self.test_args = []
-        self.test_suite = True
 
-    def run_tests(self):
-        # import here, cause outside the eggs aren't loaded
-        import pytest
-        errno = pytest.main(self.pytest_args)
-        sys.exit(errno)
-
-
-with open('README.md') as f:
+with open("README.md", encoding="utf-8") as f:
     long_description = f.read()
 
+
 setup(
-    cmdclass={'test': PyTest},
-    name='vinyldns-python',
-    version='0.9.8',
-    packages=find_packages('src'),
-    package_dir={'': 'src'},
-    py_modules=[splitext(basename(path))[0] for path in glob('src/*.py')],
+    name="vinyldns-python",
+    version="0.9.7",
+    packages=find_packages("src"),
+    package_dir={"": "src"},
```

Comment:
This changes the version from `0.9.8` to `0.9.7`, which is a regression. The version should remain `0.9.8` or be bumped higher.